logger/src/commonMain/kotlin/com/wire/kalium/logger/KaliumLogger.kt

```diff
@@ -61,96 +62,143 @@ enum class KaliumLogLevel {
     DISABLED
 }
 
+fun KaliumLogLevel.toMinSeverity(): Severity = when (this) {
+    KaliumLogLevel.VERBOSE -> Severity.Verbose
+    KaliumLogLevel.DEBUG -> Severity.Debug
+    KaliumLogLevel.INFO -> Severity.Info
+    KaliumLogLevel.WARN -> Severity.Warn
+    KaliumLogLevel.ERROR -> Severity.Error
+    KaliumLogLevel.DISABLED -> Severity.Assert
+}
+
+fun Severity.toKaliumLogLevel(): KaliumLogLevel = when (this) {
+    Severity.Verbose -> KaliumLogLevel.VERBOSE
+    Severity.Debug -> KaliumLogLevel.DEBUG
+    Severity.Info -> KaliumLogLevel.INFO
+    Severity.Warn -> KaliumLogLevel.WARN
+    Severity.Error -> KaliumLogLevel.ERROR
+    Severity.Assert -> KaliumLogLevel.DISABLED
+}
+
 /**
  * the logWriter is to create a custom writer other than the existing log writers from kermit to intercept the logs
  * in the android case we use it to write the logs on file
  *
  */
-class KaliumLogger(private val config: Config, vararg logWriters: LogWriter = arrayOf()) {
-
-    private var kermitLogger: KermitLogger
-
-    init {
-        kermitLogger = if (logWriters.isEmpty()) {
-            KermitLogger(
-                config = StaticConfig(
-                    minSeverity = config.severityLevel, listOf(platformLogWriter())
-                ),
-                tag = config.tag
-            )
-        } else {
-            KermitLogger(
-                config = StaticConfig(
-                    minSeverity = config.severityLevel, logWriters.asList()
-                ),
-                tag = config.tag
-            )
-        }
+class KaliumLogger(
+    private val config: Config = Config.DEFAULT,
+    private val tag: Tag = Tag.Text("KaliumLogger")
```

Comment:
suggestion: also here. Document what is this tag, how it behaves, etc.

Reply:
✅